src/controls/widgets/WidgetCustomUI.tsx

```diff
@@ -0,0 +1,18 @@
+import { observer } from 'mobx-react-lite'
+import { Widget_custom_componentProps_ui, Widget_custom } from 'src/controls/Widget'
+import { ImageUI } from 'src/widgets/galleries/ImageUI'
+
+export const WidgetCustomUI = observer(function WidgetCustomUI_(p: { req: Widget_custom }) {
+    const req = p.req
+
+    return (
+        <>
+            <req.customComponent value={req.componentViewState} onChange={(v) => (req.componentViewState = v)} ui={ui} />
```

Comment:
we'll probably want to use `return createElement(req.customComponent, {value:..., onChange: ... }`

[💡] and now that I see that, I just want to pass `req` directly :O 👈

Reply:
or maybe "also pass req", in addition to value, onChange, etc.

(using createElement is a good way to avoid using Fragment and adding an extra node in the virtual dom tree)